integration_tests/tests/utils/mod.rs

```diff
@@ -32,6 +32,8 @@ pub mod wallet;
 pub mod wallet_client;
 pub mod wallet_process;
 
+use tokio::time::Instant;
```

Comment:
why this change?
Did it not compile before

Reply:
Artifacts from a previous version